publishable/config/voyager.php

```diff
@@ -227,4 +227,13 @@
         //'js/custom.js',
     ],
 
+    'googlemaps' => [
+        'key'    => env('GOOGLE_MAPS_KEY'),
+        'center' => [
+            'lat' => env('GOOGLE_MAPS_DEFAULT_CENTER_LAT'),
```

Comment:
I don't think lat/lng/zoom need to pull from `env()`.  key makes perfect sense though